console/client/src/features/timeline/Timeline.tsx

```diff
@@ -96,11 +102,12 @@ export const Timeline = () => {
 
             {(() => {
               switch (entry.entry?.case) {
-                case 'call': return <TimelineCall call={entry.entry.value} />
-                case 'log': return <TimelineLog log={entry.entry.value} />
+                case 'call': return <TimelineCall call={entry.entry.value} selected={selectedEntry === entry} />
+                case 'log': return <TimelineLog log={entry.entry.value} selected={selectedEntry === entry} />
                 case 'deployment': return <TimelineDeployment
                   deployment={entry.entry.value}
                   timestamp={entry.timeStamp}
+                  selected={selectedEntry === entry}
```

Comment:
It might be good to create a `TimelineEntry` component that can hold other components. That way this selected logic could be centralized. I'll create a ticket for this: https://github.com/TBD54566975/ftl/issues/295